Resources/views/aside/folder.html.twig

```diff
@@ -0,0 +1,12 @@
+{% if treeChildItems|length > 0 %}
```

Comment:
One readability nitpick: `if treeChildItems is not empty` is more readable imho.